msi/release.sh

```diff
@@ -62,6 +147,8 @@ BUILD_EXTRA_WINPATH="$(cd "$SCRIPT_PATH"/.. && pwd -W | tr / \\\\)"
             </Component>
 EOF
 echo "$LIST" |
+sort |
+uniq |
```

Comment:
Why is this needed? AFAIR `LIST` is already sorted by virtue of being the output of the `make-file-list.sh` script...